upgrade_test.py

```diff
@@ -1244,3 +1247,166 @@ def _update_argus_upgraded_version(self, node, new_version):
                 self.log.warning("Couldn't extract version from %s", new_version)
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
+
+
+class UpgradeCustomTest(UpgradeTest):
+    def test_custom_profile_rolling_upgrade(self):
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
+        """
+        cs_user_profiles = self.prepare_data_before_upgrade()
+        self._custom_profile_rolling_upgrade(cs_user_profiles=cs_user_profiles)
+
+    def test_custom_profile_sequential_rolling_upgrade(self):
+        """
+        Run a load of a custom profile.
+        Upgrade half of nodes in the cluster, and run rollback.
+        Upgrade all nodes to new version in the end.
```

Comment:
docstring is the same as previous test. Describe what sequential means.

Reply:
fixed